js-api-doc/value/color.d.ts

```diff
@@ -1,26 +1,26 @@
 import {Value} from './index';
 
 export class SassColor extends Value {
-  static rgb(
-    red: number,
-    green: number,
-    blue: number,
-    alpha?: number
-  ): SassColor;
+  constructor(options: {
+    red: number;
+    green: number;
+    blue: number;
+    alpha?: number;
+  });
 
-  static hsl(
-    hue: number,
-    saturation: number,
-    lightness: number,
-    alpha?: number
-  ): SassColor;
+  constructor(options: {
```

Comment:
Given that the 3 overloads all have a single argument and the same return type (none as this is a constructor), the idiomatic way to write such typescript types is to use a union type instead of overloads: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#use-union-types

Reply:
As discussed elsewhere, constructor overloads are necessary here in order to disallow `new Color({hue: ..., red: ...})`.